src/components/input/examples/Inputs.vue

```diff
@@ -125,19 +125,19 @@
         autocomplete="username"
         :background="backgroundColor"
       >
-        <template slot="pre-icon">
+        <template #pre-icon>
           <cdr-icon
             use="#twitter"
           />
         </template>
-        <template slot="post-icon">
+        <template #post-icon>
```

Comment:
This use-case seems to have a tab focus problem, rendered example label "Form with Two Buttons *"
tabbing into this group places focus on the entire input, tabbing a second time does not move focus to the actual input, nor does it place a cursor into the input field, tabbing a 3rd time moves focus to the close button

Reply:
Form error has been updated to remove `tab-index=0` from the form error components